test/framework/options.py

```diff
@@ -3011,7 +3042,6 @@ def test_check_style(self):
         for pattern in patterns:
             self.assertTrue(re.search(pattern, stdout, re.M), "Pattern '%s' found in: %s" % (pattern, stdout))
 
-
```

Comment:
style nitpicking: there's supposed to be a double empty line here (always between top-level class/functions)